assets/sass/style.sass

```diff
@@ -1,9 +1,3 @@
-{{- $fonts    := site.Params.fonts }}
```

Comment:
This change is just the "poor man" attempt to fix an issue in some language I don't know.

Here's the error I'm getting with using a newer hugo on this file (keeping the loop on Params.fonts in place):

```
[root@fedora zeromq.org-f18m]# make serve
hugo server \
--buildDrafts \
--buildFuture \
--disableFastRender \
--ignoreCache
Start building sites … 
hugo v0.101.0+extended linux/amd64 BuildDate=unknown
Error: Error building site: TOCSS: failed to transform "sass/style.sass" (text/x-sass): "/storage/work/git/zeromq.org-f18m/assets/sass/style.sass:5:9": malformed URL
```

Maybe @somdoron or @sappo  you are able to do a better fix